DOC: Add notes about docker image and conda environment size (#169)
connectomicslab/connectomemapper3: docs/installation.rst

*   Open a terminal

*   Download and extract the latest release (|release|) of the BIDS App:

.. parsed-literal::

    $ docker pull sebastientourbier/connectomemapper-bidsapp:|release|
    

> .. note:: This can take a little bit of time depending on your connection speed and your machine.

Reviewer: I would say "some" instead of "a little bit"

Reviewer: same below

Author: ```suggestion
.. note:: This can take some time depending on your connection speed and your machine.
```

> .. note:: This can take a little bit of time depending on your connection speed and your machine.

Author: ```suggestion
.. note:: This can take some time depending on your connection speed and your machine.
```

        $ conda env create -f /path/to/downloaded/conda/environment[_macosx].yml

>     .. note:: This can take a little bit of time depending on your connection speed and your machine.

Author: ```suggestion
    .. note:: This can take some time depending on your connection speed and your machine.
```

>               it should take around 2.8GB of space on your machine.

Author: ```suggestion
              It should take around 2.8GB of space on your machine.
```

>               You should also expect that it would take a space of around 2.8GB on your machine.

Reviewer: For consistent wording I would say "...it would take around 2.8GB of space on your machine"

Author: ```suggestion
              it should take around 2.8GB of space on your machine.
```
